ee/codegen/src/generators/nodes/bases/base.ts

```diff
@@ -31,6 +31,7 @@ export abstract class BaseNode<
   public readonly nodeContext: V;
 
   protected readonly nodeInputsByKey: Map<string, NodeInput>;
+  protected readonly nodeInputsById: Map<string, NodeInput>;
```

Comment:
Make it easy for downstream consumers to perform node input lookups not just by key, but also by id.